policy/modules/services/xserver.te

```diff
@@ -642,6 +642,7 @@ files_watch_etc_dirs(xdm_t)
 files_watch_etc_files(xdm_t)
 files_watch_usr_dirs(xdm_t)
 files_watch_usr_files(xdm_t)
+files_watch_var_lib_dirs(xdm_t)
```

Comment:
I'd rather postpone any solution until we agree with flatpak folks how to approach this problem.
https://bugzilla.redhat.com/show_bug.cgi?id=1916652
https://bugzilla.redhat.com/show_bug.cgi?id=1928548